qa/L0_identity_cpu/test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -ex
+
```

Comment:
How about:
```
  export LD_LIBRARY_PATH=""
  export PATH=${PATH/cuda/}
```
just in case?

Reply:
I don't know if I can set `LD_LIBRARY_PATH` to empty. It's mainly because when running tritonserver image, the `LIB_LIBRARY_PATH` is set. I guess emptying it might broke something (though I'm not sure about this)

Reply:
```
export LD_LIBRARY_PATH=${LD_LIBRARY_PATH/cuda/}
``?

Reply:
hmmm, I don't fully understand also the purpose of this?

Reply:
To make sure that there is not cuda available just in case.

Reply:
OK, so this `test.sh` file is not the place to check it. Without dwelling into details about how the testing is organised, neither `setup.sh` or `test.sh` is called within the docker image containing the server.

The only way to double check CPU only mode at this stage would be to use [client API](https://github.com/triton-inference-server/client/blob/main/src/python/library/tritonclient/grpc/__init__.py) to ask the server. However, it seems, that client API doesn't expose such functionality.

Reply:
Right